spec/node/asciidoctor.spec.js

```diff
@@ -9,11 +9,20 @@ var config = {
   }
 };
 var asciidoctor = require('../../build/asciidoctor.js')(config);
+function asciidoctorVersionGreaterThan (version) {
```

Comment:
Maybe we should add an API to get the Asciidoctor version ?
Maybe we should return a double to make comparison easier ?

```
1.5.6 = 1.56
1.5.5 = 1.55
```
`.dev` version should have a low precedence than the associated normal version:

`1.5.6.dev = 1.559`